packages/fluentui/react-bindings/src/FocusZone/FocusZone.tsx

```diff
@@ -110,6 +110,7 @@ export class FocusZone extends React.Component<FocusZoneProps> implements IFocus
     preventFocusRestoration: PropTypes.bool,
     pagingSupportDisabled: PropTypes.bool,
     shouldIgnoreNotFocusable: PropTypes.bool,
+    innerRef: PropTypes.any,
```

Comment:
@layershifter I tried to use `oneOfType([PropTypes.func, PropTypes.shape({ current: PropTypes.any })])` but it was breaking in the `useAccessibility` here:

```
return React.createElement(FocusZone, {
        ...definition.focusZone.props,
        ...child.props,
        innerRef: child.ref,
        as: child.type,
        isRtl: rtl,
      });
```

because type difference

Reply:
Let's use `.any` for now